locale/en/LC_MESSAGES/django.po

```diff
@@ -3711,6 +3711,10 @@ msgstr "%(points)s points required to pass the module. "
 msgid "CHANGES_ARE_POSSIBLE_IN_EXERCISES_BEFORE_MODULE_OPENING"
 msgstr "There may be changes in the assignments before the module opens!"
 
+#: exercise/templates/exercise/_user_results.html
+msgid "CONVERT_SUBMISSION_BY_MODULE"
+msgstr "Convert Late Submissions"
```

Comment:
I'm not sure how much space is available here, but "Convert Late Submissions" may not be clear to user. Convert to what? Could "Approve Late Submissions" be better? Or if space is not an issue, "Convert Late Submissions to fully graded"

Reply:
Yes, Balancing the length and descriptiveness of the name is quite hard for me. The "Convert Late Submissions to fully graded" is the most descriptive but also too long imo. Maybe i could ask for teachers opinion on the meeting on this.